src/Umbraco.Web.UI.Client/src/views/prevalueeditors/boolean.html

```diff
@@ -1 +1 @@
-<input name="boolean" type="checkbox" ng-model="model.value" ng-true-value="1" ng-false-value="0" /> 
\ No newline at end of file
+<input name="boolean" type="checkbox" ng-model="model.value" ng-true-value="1" ng-false-value="0" id="boolean" /> <label for="boolean">True</label>
```

Comment:
@hartvig There's an html issue with this since this will exist multiple times in the DOM which means we'll have duplicate ID values for elements. For Ids, these need to be dynamic

Reply:
see ebea5e8170b4caa038a3008bc99eb9ce4b7bd530

Reply:
Great catch. Thanks!